modules/hcp-worker-hc.adoc

```diff
@@ -6,111 +6,45 @@
 [id="hcp-worker-hc_{context}"]
 = Creating worker nodes for the hosted cluster
 
-If you are working on a bare metal platform, creating worker nodes is crucial to ensure that the details in the `BareMetalHost` are correctly configured.
+If you are working on a bare metal platform, you must create worker nodes to ensure that the details in the `BareMetalHost` are correctly configured.
 
-If you are working with virtual machines, you can complete the following steps to create empty worker nodes for the Metal3 Operator to consume. To do so, you use the `kcli` tool.
+You can add Bare Metal Hosts to the cluster by using a YAML file that describes the Bare Metal Host.
```

Comment:
@jparrill @celebdor This procedure mentions a web console and states that you must be logged in as a user with cluster-admin privileges. However, we don't explicitly state where the web console is. Is the web console part of RHCOS?

Reply:
Should be part of the Openshift deployment already.